sys/auto_init/security/auto_init_atca.c

```diff
@@ -19,18 +19,46 @@
 #include "log.h"
 #include "atca.h"
 #include "atca_params.h"
+#include "kernel_defines.h"
 
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+#if (IS_ACTIVE(CONFIG_MODULE_PSA_CRYPTO))
+#include "psa_crypto_se_management.h"
+
+extern psa_drv_se_t atca_methods;
+#endif
+
 #define ATCA_NUMOF (ARRAY_SIZE(atca_params))
 
+static ATCADevice atca_devs[ATCA_NUMOF];
+
 void auto_init_atca(void)
 {
     for (unsigned i = 0; i < ATCA_NUMOF; i++) {
-        if (atcab_init((ATCAIfaceCfg *)&atca_params[i]) != ATCA_SUCCESS) {
-            LOG_ERROR("[auto_init_atca] error initializing cryptoauth device #%u\n", i);
+        atca_devs[i] = NULL;
+        int status = atcab_init_ext(&atca_devs[i], (ATCAIfaceCfg *)&atca_params[i].cfg);
+        if (status != ATCA_SUCCESS) {
+            LOG_ERROR("[auto_init_atca] error initializing cryptoauth device #%u, status: 0x%02x\n",
+                      i, status);
+            continue;
+        }
+#if (IS_ACTIVE(CONFIG_MODULE_PSA_CRYPTO))
+        if (i >= PSA_MAX_SE_COUNT) {
+            LOG_ERROR("[auto_init_atca] PSA Crypto – too many secure elements #%u\n", i + 1);
+            continue;
+        }
+
+        DEBUG("Registering Driver with location: %lx\n", atca_params[i].atca_loc);
+        status = psa_register_secure_element(atca_params[i].atca_loc, &atca_methods, atca_devs[i]);
+        if (status != PSA_SUCCESS) {
+            LOG_ERROR(
+                "[auto_init_atca] PSA Crypto – error registering cryptoauth PSA driver \
+                for device #%u, status: %d\n", i,
+                status);
```

Comment:
i don't know where the static checker found a "comma preceded by whitespace" but this is better a little less spacy
 
```suggestion
            LOG_ERROR("[auto_init_atca] PSA Crypto – error registering cryptoauth PSA driver\
                       for device #%u, status: %d\n", i,status);
```